src/overlays/actors/ovl_En_Syateki_Itm/z_en_syateki_itm.c

```diff
@@ -20,21 +30,319 @@ const ActorInit En_Syateki_Itm_InitVars = {
     (ActorFunc)EnSyatekiItm_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Syateki_Itm/EnSyatekiItm_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Syateki_Itm/EnSyatekiItm_Destroy.s")
+Vec3f D_80B10240 = { 0.0f, -10.0f, -270.0f };
+Vec3f D_80B1024C = { -220.0f, 66.0f, -320.0f };
+Vec3f D_80B10258 = { 260.0f, 66.0f, -320.0f };
+Vec3f D_80B10264 = { 0.0f, -10.0f, -270.0f };
+Vec3f D_80B10270 = { -220.0f, 66.0f, -320.0f };
+Vec3f D_80B1027C = { 260.0f, 66.0f, -320.0f };
```

Comment:
if these are all going to end up being in function static, you can define them in the else of the non_matching block and put them inside now

Reply:
Unfortunately they cannot. sRupeeTypes is used in Init.